### PR TITLE
Release 2022.0.2 kit image version

### PR DESCRIPTION
**Pull request by trickeydan**
Also updates the library list.

Changelog:

- Updated marker size to 200mm
- Added user to `dialout` group
- Added extra USB PID for Ruggeduino

### CODE REVIEW

File: programming/python/libraries.md [hunk 1]
```diff
@@ -22,30 +22,31 @@ The following python libraries are installed and available for use in your robot
...
 * [rpm 4.16.1.3](https://pypi.org/project/rpm)
 * [setuptools 57.4.0](https://pypi.org/project/setuptools)
 * [six 1.16.0](https://pypi.org/project/six)
-* [sr-robot3 2022.0.1](https://pypi.org/project/sr.robot3/)
+* [sr-robot3 2022.0.2](https://pypi.org/project/sr.robot3/)
 * [toml 0.10.2](https://pypi.org/project/toml)
 * [typing-extensions 3.10.0.0](https://pypi.org/project/typing-extensions)
 * [udiskie 2.3.3](https://pypi.org/project/udiskie)
-* udiskie-locale-en-us 2.3.3
+* python3-udiskie-locale-en-us 2.3.3
```
Comment (RealOrangeOne) [inline programming/python/libraries.md:49]: Question: Is the `python3-` prefix here required?

**Comment by PeterJCLaw** [comment]
Why is the brand submodule changing here?

**Comment by raccube** [comment]
Probably because it's very easy to accidentally update

File: programming/python/libraries.md (hunk 1 above)
Comment (trickeydan) [reply programming/python/libraries.md]: Nope

Comment (PeterJCLaw) [reply programming/python/libraries.md]: Does this apply to `pkg-resources` too?

File: _data/kit_versions.yml [hunk 2]
```diff
@@ -1,2 +1,4 @@
+- version: 2022.0.1
+  release: 2021-11-20
```
Comment (PeterJCLaw) [inline _data/kit_versions.yml:2]:
```suggestion
  released: 2021-11-20
```

Comment (trickeydan) [reply _data/kit_versions.yml]: Good spot! We should consider validating this file in CI.

**Review by PeterJCLaw — APPROVED** [review]

**Pull request merged**